<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrefInput</class>
 <widget class="QWidget" name="PrefInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>549</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="input_tab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>&amp;Keyboard</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="keyboard_icon">
           <property name="text">
            <string notr="true">icon</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="actioneditor_desc">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Here you can change any key shortcut. To do it double click or press enter over a shortcut cell. Optionally you can also save the list to share it with other people or load it in another computer.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ActionsEditor" name="actions_editor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QCheckBox" name="globalshortcuts_check">
           <property name="text">
            <string>&amp;Use the multimedia keys as global shortcuts</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="globalshortcuts_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Select &amp;keys...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>&amp;Mouse</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" rowspan="8">
        <layout class="QVBoxLayout">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="mouse_icon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string notr="true">icon</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="textLabel1_16">
           <property name="text">
            <string>Button functions:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="textLabel2_6">
           <property name="text">
            <string>&amp;Left click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>left_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="MyComboBox" name="left_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="textLabel3_4">
           <property name="text">
            <string>&amp;Double click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>double_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="MyComboBox" name="double_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="textLabel3_5">
           <property name="text">
            <string>&amp;Right click</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>right_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="MyComboBox" name="right_click_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>M&amp;iddle click</string>
           </property>
           <property name="buddy">
            <cstring>middle_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="middle_click_combo"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>X Button &amp;1</string>
           </property>
           <property name="buddy">
            <cstring>xbutton1_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="xbutton1_click_combo"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>X Button &amp;2</string>
           </property>
           <property name="buddy">
            <cstring>xbutton2_click_combo</cstring>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="xbutton2_click_combo"/>
         </item>
         <item row="0" column="2" rowspan="7">
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <widget class="Line" name="line7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::HLine</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QLabel" name="textLabel2_8">
           <property name="text">
            <string>&amp;Wheel function:</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="buddy">
            <cstring>wheel_function_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="wheel_function_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Dra&amp;g function:</string>
           </property>
           <property name="buddy">
            <cstring>drag_function_combo</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="drag_function_combo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="1">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QCheckBox" name="delay_left_check">
         <property name="text">
          <string>Don't &amp;trigger the left click action with a double click</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="wheel_function_seeking_reverse_check">
         <property name="text">
          <string>R&amp;everse wheel media seeking</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>128</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>M&amp;ouse wheel functions</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="wheel_functions_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Select the actions that should be cycled through when using the &quot;Change function of wheel&quot; option.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="wheel_function_seek">
         <property name="text">
          <string>Media &amp;seeking</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="wheel_function_volume">
         <property name="text">
          <string>&amp;Volume control</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="wheel_function_zoom">
         <property name="text">
          <string>&amp;Zoom video</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="wheel_function_speed">
         <property name="text">
          <string>&amp;Change speed</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
  <customwidget>
   <class>ActionsEditor</class>
   <extends>QWidget</extends>
   <header>actionseditor.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>input_tab</tabstop>
  <tabstop>globalshortcuts_check</tabstop>
  <tabstop>globalshortcuts_button</tabstop>
  <tabstop>left_click_combo</tabstop>
  <tabstop>double_click_combo</tabstop>
  <tabstop>right_click_combo</tabstop>
  <tabstop>middle_click_combo</tabstop>
  <tabstop>xbutton1_click_combo</tabstop>
  <tabstop>xbutton2_click_combo</tabstop>
  <tabstop>wheel_function_combo</tabstop>
  <tabstop>drag_function_combo</tabstop>
  <tabstop>delay_left_check</tabstop>
  <tabstop>wheel_function_seeking_reverse_check</tabstop>
  <tabstop>wheel_function_seek</tabstop>
  <tabstop>wheel_function_volume</tabstop>
  <tabstop>wheel_function_zoom</tabstop>
  <tabstop>wheel_function_speed</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>globalshortcuts_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>globalshortcuts_button</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>509</x>
     <y>526</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
